marconi-sidechain/src/Marconi/Sidechain/Error.hs

```diff
@@ -16,3 +29,48 @@ data QueryExceptions
   | IndexerInternalError !Text
   deriving stock (Show)
   deriving anyclass (Exception)
+
+-- * Mapping of exceptions to exit codes
+
+class HasExitCode e where
+  toExitCode :: e -> Int
+
+instance HasExitCode (IndexerError Void) where
+  toExitCode = indexerErrorToExit
```

Comment:
Same here, `Void` can be replaced by `a`.

Reply:
Did you mean to add another comment at a different point where I've used `Void`? I'm assuming you mean line 41 of this file, but I just want to double check.